### PR TITLE
Mark node as loading in Cluster pipeline read

### PR DESCRIPTION
**Pull request by vmihailenco**
Fixes https://github.com/go-redis/redis/issues/1031

### CODE REVIEW

File: cluster.go [hunk 1]
```diff
@@ -1312,6 +1312,10 @@ func (c *ClusterClient) pipelineProcessCmds(
 	err = cn.WithReader(c.opt.ReadTimeout, func(rd *proto.Reader) error {
 		return c.pipelineReadCmds(node, rd, cmds, failedCmds)
 	})
+	if err == nil {
```
Comment (theodesp) [inline cluster.go]: If you return `err` anyway why don't you return it as it is?

File: go.mod [hunk 2]
```diff
@@ -4,3 +4,5 @@ require (
 	github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega v1.5.0
 )
+
+go 1.13
```
Comment (theodesp) [inline go.mod]: We don't really need that info as nothing changed here

File: cluster.go (hunk 1 above)
Comment (vmihailenco) [reply cluster.go]: Fixed

File: go.mod (hunk 2 above)
Comment (vmihailenco) [reply go.mod]: This is what go mod does automatically - not going to fight it on this matter

**Pull request merged**